docs/book/third_party/crate_install.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+# From https://github.com/japaric/trust/blob/2d08818370e618108b55fae9b243330640554dce/install.sh
```

Comment:
Let's have this script to be an unmodified copy of https://github.com/japaric/trust/blob/gh-pages/install.sh if possible, and include the LICENSE file https://github.com/japaric/trust/blob/master/LICENSE-APACHE

The directory structure for third party should be:
`/third_party/japaric-trust/...` (in the top level repo, where we already have other third party code

That should do it?

Reply:
If it's unmodified, we'll need to put the target determination logic either in the makefile, or in a new script that determines the target, then calls crate_install.sh. If that's too much of a pain, we can keep things the way they are now, with the edits to the script.

Reply:
Oh, true. I guess I assumed we wanted to leave the target unset so that developers on OSX and Linux would get the right binary, but we could wrap the script in another script that has the PR's logic to select the right target without rustc.

Reply:
That all said, I think I prefer just modifying the script, it seems simpler.